docs/csharp/programming-guide/unsafe-code-pointers/pointer-types.md

```diff
@@ -1,140 +1,96 @@
-  
- A pointer can be `null`. Applying the indirection operator to a null pointer causes an implementation-defined behavior.  
-  
- Be aware that passing pointers between methods can cause undefined behavior. Consider a method that returns a pointer to a local variable through an `in`, `out` or `ref` parameter or as the function result. If the pointer was set in a fixed block, the variable to which it points may no longer be fixed.  
-  
- The following table lists the operators and statements that can operate on pointers in an unsafe context:  
-  
-|Operator/Statement|Use|  
-|-------------------------|---------|  
-|*|Performs pointer indirection.|  
-|->|Accesses a member of a struct through a pointer.|  
-|[]|Indexes a pointer.|  
-|`&`|Obtains the address of a variable.|  
-|++ and --|Increments and decrements pointers.|  
-|+ and -|Performs pointer arithmetic.|  
-|==, !=, \<, >, \<=, and >=|Compares pointers.|  
-|`stackalloc`|Allocates memory on the stack.|  
-|`fixed` statement|Temporarily fixes a variable so that its address may be found.|  
-  
-## C# Language Specification  
- [!INCLUDE[CSharplangspec](~/includes/csharplangspec-md.md)]  
-  
-## See Also  
- [C# Programming Guide](../../../csharp/programming-guide/index.md)  
- [Unsafe Code and Pointers](../../../csharp/programming-guide/unsafe-code-pointers/index.md)  
- [Pointer Conversions](../../../csharp/programming-guide/unsafe-code-pointers/pointer-conversions.md)  
- [Pointer Expressions](../../../csharp/programming-guide/unsafe-code-pointers/pointer-expressions.md)  
- [Types](../../../csharp/language-reference/keywords/types.md)  
- [unsafe](../../../csharp/language-reference/keywords/unsafe.md)  
- [fixed Statement](../../../csharp/language-reference/keywords/fixed-statement.md)  
- [stackalloc](../../../csharp/language-reference/keywords/stackalloc.md)  
- [Boxing and Unboxing](../../../csharp/programming-guide/types/boxing-and-unboxing.md)
+
+In an unsafe context, a type may be a pointer type, a value type, or a reference type. A pointer type declaration takes one of the following forms:
+
+``` csharp
+type* identifier;
+void* identifier; //allowed but not recommended
+```
+
+Any of the following types may be a pointer type:
+
+- [sbyte](../../language-reference/keywords/sbyte.md), [byte](../../language-reference/keywords/byte.md), [short](../../language-reference/keywords/short.md), [ushort](../../language-reference/keywords/ushort.md), [int](../../language-reference/keywords/int.md), [uint](../../language-reference/keywords/uint.md), [long](../../language-reference/keywords/long.md), [ulong](../../language-reference/keywords/ulong.md), [char](../../language-reference/keywords/char.md), [float](../../language-reference/keywords/float.md), [double](../../language-reference/keywords/double.md), [decimal](../../language-reference/keywords/decimal.md), or [bool](../../language-reference/keywords/bool.md).
```

Comment:
I would break this list up into several additional bulleted list items to make the individual type categories more visible:
- any integral type (with a list of them here)
- any floating point type (float and double)
- the char type
- the bool type